object/types.proto

```diff
@@ -8,126 +8,77 @@ option csharp_namespace = "NeoFS.API.Object";
-
-    // integrity carries object integrity evidence.
-    Integrity integrity = 1;
-
-    // Attribute groups the parameters of the object attributes.
-    message Attribute {
-      // key carries the string key to the object attribute.
-      string key = 1;
-
-      // value carries the string value of the object attribute.
-      string value = 2;
-    }
-
-    // attributes carries list of the object attributes in a string key-value format.
-    repeated Attribute attributes = 2;
-
-    // creation_epoch carries number of NeoFS epoch on which the object was created.
-    uint64 creation_epoch = 3;
-
-    // Tombstone groups the options for deleting an object.
-    message Tombstone {
-    }
-
-    // Tombstone marks the object to be deleted.
-    Tombstone tombstone = 4;
-
-    // homomorphic_hash carries homomorphic hash of the object payload.
-    bytes homomorphic_hash = 5;
-
-    // StorageGroup groups meta information about a storage group.
-    message StorageGroup {
-    }
-
-    // StorageGroup marks an object containing information about a storage group.
-    StorageGroup storage_group = 6;
-
-    // Split groups information about spawning the object through a payload
-    // splitting.
-    message Split {
-      // Parent carries identifier of the origin object.
-      refs.ObjectID parent = 1;
-
-      // Previous carries identifier of the left split neighbor.
-      refs.ObjectID previous = 2;
-
-      // Next carries identifier of the right split neighbor.
-      refs.ObjectID next = 3;
-
-      // Children carries list of identifiers of the objects generated by
-      // splitting the current.
-      repeated refs.ObjectID children = 4;
-
-      // Origin carries the header of the origin object.
-      Header origin = 5;
-    }
-
-    // Split carries the position of the object in the split hierarchy.
-    Split split = 7;
+  ObjectType object_type = 7;
+  // Homomorphic hash of the object payload.
+  bytes homomorphic_hash = 8;
+  // Session token, if it was used during Object creation.
+  // Need it to verify integrity and authenticity out of Request scope.
+  service.SessionToken session_token = 9;
+
+  // Attribute groups the user-defined Key-Value pairs attached to the object
+  message Attribute {
+    // string key to the object attribute
+    string key = 1;
+    // string value of the object attribute
+    string value = 2;
   }
-
-  // Extended carries the additional part of the header.
-  Extended extended = 2;
+  repeated Attribute attributes = 10;
+
+  // Information about spawning the objects through a payload splitting.
+  message Split {
+    // Identifier of the origin object.
+    // Parent and children objects must be within the same container.
+    // Parent object_id id known only to the minor child.
+    refs.ObjectID parent = 1;
+    // Previous carries identifier of the left split neighbor.
+    refs.ObjectID previous = 2;
+    // `signature` field of the parent object. Used to reconstruct parent.
```

Comment:
Isn't it signature of `parent_header` then?

Reply:
Technically it is, in the same was as ObjectID.